dali/operators/reader/loader/nemo_asr_loader.h

```diff
@@ -110,10 +143,13 @@ class DLL_PUBLIC NemoAsrLoader : public Loader<CPUBackend, AsrSample> {
   float quality_;
   bool downmix_;
   DALIDataType dtype_;
-  float max_duration_ = 0.0;
-  bool normalize_text_ = false;
+  float max_duration_;
+  bool normalize_text_;
 
+  int num_threads_;
+  ThreadPool thread_pool_;
```

Comment:
So we have a separate, dedicated thread pool here.

Reply:
SO we can have 2 the number of threads the user requested working in parallel, one set in the Nemo reader, and another set in CPU ops.